docs/content/docs/ops/state/task_failure_recovery.md

```diff
@@ -334,7 +334,7 @@ A region is a set of tasks that communicate via pipelined data exchanges.
 That is, batch data exchanges denote the boundaries of a region.
 - All data exchanges in a DataStream job or Streaming Table/SQL job are pipelined.
 - All data exchanges in a Batch Table/SQL job are batched by default.
-- The data exchange types in a DataSet job are determined by the 
+- The data exchange types in a DataStream job are determined by the 
```

Comment:
I think this page is outdated. Currently, all DataStream / Table / SQL jobs should rely on `ExecutionMode` for determining the data exchange type. We probably can combine these 3 items into 1.

Reply:
Sure. Merged.